backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveDiskCommand.java

```diff
@@ -170,10 +170,18 @@ protected Map<String, Pair<String, String>> getExclusiveLocks() {
 
```

Comment:
This commit reverts an issue introduced in the same branch in a previous commit, it would be better to just drop the commit that introduced the wrong change, or squash them together (since it includes other changes) and add a proper commit message.

Also, the sha in the current commit message is outdated.